bower.json

```diff
@@ -1,27 +1,54 @@
 {
-    "name": "butter",
-    "dependencies": {
-        "marionette": "^2.4.1",
-        "bootstrap": "~3.3.7",
-        "video.js": "4.11.4",
-        "videojs-youtube": "1.2.10",
-        "mousetrap": "~1.6.0",
-        "font-awesome": "^4.6.3"
-    },
-    "dependenciesIgnore": {
-        "backbone": ["**/!(backbone-min.js)"],
-        "jquery": ["**/!(dist)"],
-        "font-awesome": ["less", "scss", ".*"],
-        "video.js": ["**/!(dist)"],
-        "videojs-youtube": ["**/!(src)"],
-        "underscore": ["**/!(underscore-min.js)"],
-        "mousetrap": ["tests", ".gitignore", "./*!mousetrap.min.js"],
-        "marionette": ["**/!(lib)"],
-        "backbone.babysitter": ["**/!(lib)"],
-        "backbone.wreqr": ["**/!(lib)"],
-        "bootstrap": ["**/!(dist)"]
-    },
-    "resolutions": {
-        "underscore": ">=1.8.0"
-    }
+  "name": "butter",
+  "dependencies": {
+    "marionette": "^2.4.1",
+    "bootstrap": "~3.3.7",
+    "video.js": "4.11.4",
+    "videojs-youtube": "1.2.10",
+    "mousetrap": "~1.6.0",
+    "font-awesome": "^4.6.3",
+    "flag-icon-css": "^2.7.0"
```

Comment:
that's a mistake, clearly, i'd like to have it in npm, but i'm not sure how to include it...